pkg/cmd/lint-epic-issue-refs/lint_epic_issue_refs.go

```diff
@@ -0,0 +1,319 @@
+// warning that it is not a common case and to check it is intended.
+//
+// Returns:
+// - True if the check passed. False if it failed.
+func checkMultipleEpicsInCommits(commitInfos []commitInfo, params *parameters) bool {
+	passedCheck := true
+	for _, ci := range commitInfos {
+		if len(ci.epicRefs) > 1 {
+			fmt.Printf(
+				"Warning: This commit references multiple epics. Normally a commit only references one epic. "+
+					"Noting this to verify this commit relates to multiple epics. "+
+					"epic_refs=[%v], commit: %s\n\n",
+				ci.epicRefs,
+				commitShaURL(ci.sha, params),
+			)
+			passedCheck = false
+		}
+	}
+
+	return passedCheck
+}
+
+func lintEpicAndIssueRefs(ctx context.Context, ghClient *github.Client, params *parameters) error {
+	commitInfos, err := scanCommitsForEpicAndIssueRefs(ctx, ghClient, params)
+	if err != nil {
+		return err
+	}
+
+	pr, _, err := ghClient.PullRequests.Get(
+		ctx, params.Org, params.Repo, params.PrNumber,
+	)
+	if err != nil {
+		// nolint:errwrap
+		return fmt.Errorf("error getting pull requests from GitHub: %v", err)
+	}
+	prBody := pr.GetBody()
+
+	// Skip checking multi-PR backports for now. Verifying them is complex and the source PRs
+	// have already been checked.
+	if isMultiPrBackport(prBody) {
+		// TODO(jamesl): handle backport PRs containing multiple source PRs differently
+		// How might they be handled?
+		// - check that all source PR bodies have epic or issue refs?
+		// - figure out which commits in this PR correspond to the commits in the source PRs and ...
+		// - check all source PRs have all the expected references a PR should have. Do something
+		//   with that info, like maybe link it back to the commits in this PR and report on it
+		//   somehow??? (This seems really heavy / not useful.)
+		// - don't check backports with multiple source PRs for the complex cases like
+		//   "checkPrEpicsUsedInCommits"?
+		// - these checks are really for making it possible for the automated docs issue generation to
+		//   get the right epic(s) applied to them. Maybe make this a super simple check and then add
+		//   info to the generated docs issue stating it needs to be reviewed more. This case shouldn't
+		//   happen too often. The majority of backports are for a single source PR.
+		fmt.Println("This is a multi-PR backport. Skipping checking it.")
+		return nil
+	}
+
+	var prInfo = prBodyInfo{
+		epicRefs:        extractEpicIDs(prBody),
+		epicNone:        extractEpicNone(prBody),
+		issueCloseRefs:  extractFixIssueIDs(prBody),
+		issueInformRefs: extractInformIssueIDs(prBody),
+		prNumber:        params.PrNumber,
+	}
+
+	passedAllTests := checkForMissingRefs(prInfo, commitInfos, params) &&
+		checkPrEpicsUsedInCommits(prInfo, commitInfos, params) &&
+		checkMultipleEpicsInCommits(commitInfos, params)
+
+	if !passedAllTests {
+		fmt.Println("See also: https://wiki.crdb.io/wiki/spaces/CRDB/pages/2009039063/")
```

Comment:
```suggestion
        fmt.Println("For more information about issue and epic references, see: https://wiki.crdb.io/wiki/spaces/CRDB/pages/2009039063/")
```

Suggesting just the text change. The GitHub UI won't allow entry of tabs.

Reply:
Done.